libkbfs/key_bundle_v3.go

```diff
@@ -25,6 +26,28 @@ import (
 // key information.
 type DeviceKeyInfoMapV3 map[kbfscrypto.CryptPublicKey]TLFCryptKeyInfo
 
+// Size implements the cache.Measurable interface.
+func (dkimV3 DeviceKeyInfoMapV3) Size() int {
+	// statically-sized part
+	mapSize := cache.StaticSizeOfMap(
```

Comment:
Can we make this `StaticSizeOfMapWithSize` instead and do the reflect only once and save it to a const?  Same below.